recipes/scheduler.rb

```diff
@@ -15,11 +12,39 @@
 template node['snapraid']['scheduler']['config_file'] do
 end
 
-# Setup our cronjob
-cron 'snapraid-runner' do
-  hour node['snapraid']['scheduler']['execution_hour']
-  minute node['snapraid']['scheduler']['execution_minute']
-  command "sudo python #{node['snapraid']['scheduler']['script_directory']}/snapraid-runner.py "\
-    "-c #{node['snapraid']['scheduler']['config_file']}"
-  user 'root'
+#
+# Install Python & configure scheduled task (Windows)
+#
+if platform?('windows')
+  windows_package 'python2.7' do
```

Comment:
Yeah I did debate that when I introduced it originally, but seem to remember that the python cookbook was unsuitable for some reason. I forget what it was. I'll raise it as an issue to fix later :)